platform/default/src/mbgl/storage/offline_database.cpp

```diff
@@ -1136,4 +1133,16 @@ bool OfflineDatabase::exceedsOfflineMapboxTileCountLimit(const Resource& resourc
         && offlineMapboxTileCountLimitExceeded();
 }
 
+void OfflineDatabase::clearCache() {
```

Comment:
should it notify about success / failure?

Reply:
Fixed